### PR TITLE
#1269 RbRelease for receiving Github release event

### PR DESCRIPTION
**Pull request by carlosmiranda**
#1269: Added `RbRelease` to receive Github [release event](https://developer.github.com/v3/activity/events/types/#releaseevent). This `Rebound` implementation responds to `published` Github action, and generates a claim `Release was published`.

Added puzzle for further implementation of stakeholder that responds to the claim in order to pay the [architect release bonus](http://www.zerocracy.com/policy.html#53).

### CODE REVIEW

**Comment by 0crat** [comment]
Job [#1355](https://github.com/zerocracy/farm/issues/1355) is now in scope, role is `REV`

**Comment by 0crat** [comment]
This pull request [#1355](https://github.com/zerocracy/farm/issues/1355) is assigned to @krzyk[/z](https://www.0crat.com/u/krzyk), here is [why](https://www.0crat.com/footprint/C3NDPUA8L/1870872561); the budget is 15 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§27](http://www.zerocracy.com/policy.html#27) and when you decide to accept the changes, inform @g4s8[/z](https://www.0crat.com/u/g4s8) (the architect) right in this ticket; if you decide that this PR should not be accepted ever, also inform the architect; this [blog post](http://www.yegor256.com/2015/02/09/serious-code-reviewer.html) will help you understand what is expected from a code reviewer; there will be a monetary reward for this job

File: src/test/java/com/zerocracy/radars/github/RbReleaseTest.java [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+import javax.json.Json;
+import org.hamcrest.MatcherAssert;
+import org.hamcrest.Matchers;
+import org.junit.Test;
+
+/**
+ * Test case for {@link RbRelease}.
+ * @author Carlos Miranda (miranda.cma@gmail.com)
+ * @version $Id$
+ * @since 0.25
```
Comment (krzyk) [inline src/test/java/com/zerocracy/radars/github/RbReleaseTest.java]: @carlosmiranda release `0.26`

File: src/main/java/com/zerocracy/radars/github/RbRelease.java [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+        final String answer;
+        if (event.containsKey(RbRelease.JSON_KEY)) {
+            final JsonObject release = event.getJsonObject(RbRelease.JSON_KEY);
+            new ClaimOut()
+                .type("Release was published")
+                .param("tag", release.getString("tag_name"))
+                .param("date", release.getString("published_at"));
+            answer = String.format(
+                "Release published: %d",
+                release.getInt("id")
```
Comment (krzyk) [inline src/main/java/com/zerocracy/radars/github/RbRelease.java]: @carlosmiranda I'm not sure if `id` of release is useful in itself, I would prefer to see also tag here.

**Comment by carlosmiranda** [comment]
@krzyk updated, please check

**Comment by krzyk** [comment]
@carlosmiranda thanks

**Comment by krzyk** [comment]
@rultor merge

**Comment by g4s8** [comment]
@rultor merge

**Comment by 0crat** [comment]
@ypshenychka[/z](https://www.0crat.com/u/ypshenychka) please review this job completed by @krzyk[/z](https://www.0crat.com/u/krzyk), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by 0crat** [comment]
The job [#1355](https://github.com/zerocracy/farm/issues/1355) is now out of scope

**Comment by 0crat** [comment]
Payment to `ARC` for a closed pull request, as in [§28](http://www.zerocracy.com/policy.html#28): +10 point(s) just awarded to @g4s8[/z](https://www.0crat.com/u/g4s8)

**Comment by ypshenychka** [comment]
@0crat quality good

**Comment by 0crat** [comment]
Order was finished, quality is "good": +23 point(s) just awarded to @krzyk[/z](https://www.0crat.com/u/krzyk)

**Comment by 0crat** [comment]
Quality review completed: +8 point(s) just awarded to @ypshenychka[/z](https://www.0crat.com/u/ypshenychka)

**Pull request merged**